test-java-lib-lombok/src/main/java/com/uber/lombok/LombokDTO.java

```diff
@@ -36,4 +36,5 @@ public class LombokDTO {
   private String field;
   @Builder.Default private String fieldWithDefault = "Default";
   @Nullable private String nullableField;
+  @Nullable @Builder.Default private String fieldWithNullDefault = null;
```

Comment:
So before this PR, this line would result in a NullAway error?  Just want to make sure we have the right NullAway config to test the fix here

Reply:
Yes, though the error would be reported at the class level, since it actually happens in a new generated method of `LombokDTO`.

Reply:
Yikes, glad we're getting rid of this awful error message!